src/modules/auth/googleauth/dto/profile.dto.ts

```diff
@@ -0,0 +1,18 @@
+interface Emails {
```

Comment:
you need negotiate in the team about dto naming, as I saw that guys put Dto in the end, like EmailsDto, I like this approach better as it help to understand that data came from client

Reply:
fixed